### ISSUE TITLE
Modifications not made upstream

### ISSUE THREAD

**Issue by doctormo** [a-dekker/sailcron#7]
It appears like you have modified cron-descriptor and python-crontab, but the changes have not been made upstream. So they won't help others and the fork will become more expensive for sailcron over time.

Were there any changes to python-crontab that I could take upstream for you?

**Comment by a-dekker** [comment]
I encountered serveral issues with cron-descriptor, which where swiftly fixed upstream after reporting. Apart from that, I included a few additional lines which look specific to the OS of this app (another locale import was needed).

As for python-crontab (crontab.py) I am not aware of any changes I made. I only use a small portion for cron validation, which works very well. There is a small issue btw, e.g. weekdays 3-1 is accepted but not valid. I did not find a github repo for python-crontab (and do not have a launchpad account), so I didn't report yet.

So I have no intention to create my own versions and want to report issues/make pull requests upstream if relevant. The reason I included crontab.py here is purely practical, so it is easy to package. I could miss upstream fixes, but I am watching the cron-descriptor repository just in case.
